### DISCUSSION TITLE
Use of Access Bit

### DISCUSSION THREAD

**Discussion by kennyschuoler** [SmartEVSE/SmartEVSE-2#28] [Q&A] [answered]
Hi @bobosch , @mstegen ,

What is the desired use for the access bit, is it the purpose to enable/disable charging in general, or only in combination with e.g. the RFID ?
I currently don't use RFID or any other access verification , but i do use the access bit in my automation to disable charging at dynamic moments (e.g. i first let my house battery fully charge on solar, only then i enable access bit ( solar is automatically set at 7h in de morning).
If this is not the intended use, then i will have a look and introduce something similar. If it is the intended use, then i will verify if everything works as designed ( e.g. switching from smart to solar "resets" the access bit)

Thanks !

**Comment by mstegen** [comment, marked as answer]
The access bit is used for enable/disable the EVSE. 
It can be enabled/ disabled with:
- An external switch, connected to the SW input and ground. 
- The RFID reader
- a modbus command (like you used)
